tests/chainermn_tests/functions_tests/test_point_to_point_communication.py

```diff
@@ -12,6 +12,13 @@
 import chainermn.functions
 
 
+class Param(object):
+    def __init__(self):
+        self.x = None
+        self.model = None
+        self.entire_model = None
+
+
 class TestPointToPointCommunication(unittest.TestCase):
 
     def setup(self, gpu):
```

Comment:
This setup is too large. Could you split using `pytest.mark.parametrize` ?
Please refer to the following link.
https://github.com/chainer/chainer/blob/master/tests/chainermn_tests/communicator_tests/test_communicator.py#L305

Reply:
It seems `parametrize` does not work with `unittest`[link](https://github.com/pytest-dev/pytest/issues/3484)
If we want to do that, we need to remove the `unittest`